dask_bigquery/core.py

```diff
@@ -206,19 +204,7 @@ def make_create_read_session_request():
         )
         meta = schema.empty_table().to_pandas(**arrow_options)
 
-        label = "read-gbq-"
-        output_name = label + tokenize(
-            project_id,
-            dataset_id,
-            table_id,
-            row_filter,
-            read_kwargs,
-        )
-
-        layer = DataFrameIOLayer(
-            output_name,
-            meta.columns,
-            [stream.name for stream in session.streams],
+        return dd.from_map(
```

Comment:
Could you pass the label into from_map to make the task prefix more descriptive?

Reply:
Done